main.py

```diff
@@ -40,7 +42,7 @@
 
 
 class Settings:
-    do_update_wikidata = True
+    do_update_wikidata = False
```

Comment:
This change was committed by accident. Will revert it. I think we should get these setting also out in the new settings-file.